src/inverseRobot.js

```diff
@@ -24,7 +24,19 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inversedRobot = {};
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    if (inversedRobot.hasOwnProperty(value)) {
```

Comment:
Something wrong with naming, cuz hasOwnProperty checks `keys` but not the values
```suggestion
    if (inversedRobot.hasOwnProperty(value)) {
```